### ISSUE TITLE
OperaDriver is no longer part of Selenium

### ISSUE THREAD

**Issue by alan-morey** [bonigarcia/selenium-jupiter#313]
**Description of the problem**: Selenium removed OperaDriver in Selenium 4.5 and now selenium-jupiter no longer compiles.

```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/alan.morey/repos/selenium-jupiter/src/main/java/io/github/bonigarcia/seljup/CapabilitiesHandler.java:[48,33] package org.openqa.selenium.opera does not exist
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  12.235 s
[INFO] Finished at: 2022-10-28T11:50:26-07:00
[INFO] ------------------------------------------------------------------------                                                                                                           
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project selenium-jupiter: Compilation failure                              
[ERROR] /home/alan.morey/repos/selenium-jupiter/src/main/java/io/github/bonigarcia/seljup/CapabilitiesHandler.java:[48,33] package org.openqa.selenium.opera does not exist
```

Looks like build has been failing since dependency was upgraded on Sept 28: https://github.com/bonigarcia/selenium-jupiter/actions/runs/3148987861

**Comment by bonigarcia** [comment]
Thanks for reporting. It is fixed now. I'll try to make a new release of Selenium-Jupiter soon.

**Comment by bonigarcia** [comment]
This is fixed in Selenium-Jupiter 4.3.2, just released.
